src/locale/dim.json

```diff
@@ -374,6 +374,7 @@
     },
     "HideAllConfigs": "Hide all configurations",
     "HideConfigs": "Hide configurations",
+    "IncompatibleWithOptimizer": "This item is incompatible with the optimizer. Please purchase a new version from collections.",
```

Comment:
Optimizer & Collections should be capitalized, and the game/help consistently calls it "reacquire"

Reply:
Ok will update

Reply:
Done